.busted

```diff
@@ -0,0 +1,9 @@
+return {
```

Comment:
We are currently sending the options in the `script/test` file, but I like the idea of using this file for clarity.
Apart from that, is the `lpath` option needed?